### PR TITLE
Change values from `0.001` to `0.1` / use config from `btrack-examples`

### LINKED ISSUE #372: [FEATURE REQUEST] Set default value of p(not track)
should be: 0.1

### PR DESCRIPTION
**Pull request by paddyroddy**
Fixes #372

### CODE REVIEW

**Review by quantumjot — APPROVED** [review]
👌

File: models/cell_config_flat.json
Comment (paddyroddy) [inline models/cell_config_flat.json:1]: Apparently this should never have been here

Comment (paddyroddy) [reply models/cell_config_flat.json:1]: @p-j-smith 👋

**Comment by p-j-smith** [comment]
I think the changes will need to be made in the example configs (https://github.com/lowe-lab-ucl/btrack-examples) rather than here, as the defaults are essentially overridden and set using the example `cell_config.json`

**Review by p-j-smith — APPROVED** [review]
nice! I think this will clear up some confusion

File: .gitignore [hunk 1]
```diff
@@ -55,8 +55,7 @@ instance
 lib
 lib64
 local_settings.py
-models/MDCK_*
-models/test_config.json
+models
```
Comment (paddyroddy) [inline .gitignore:58]: We don't need this folder at all now

File: tests/test_config.py [hunk 2]
```diff
@@ -87,9 +87,9 @@ def _cfg_dict() -> tuple[dict, dict]:
     return cfg, cfg
 
 
-def _cfg_file() -> tuple[Path, dict]:
+def _cfg_file() -> tuple[str, dict]:
     filename = CONFIG_FILE
-    assert isinstance(filename, Path)
+    assert isinstance(filename, str)
```
Comment (paddyroddy) [inline tests/test_config.py:92]: This test seems a bit stupid. But I'll leave it here for now

**Comment by paddyroddy** [comment]
PR changes purpose a bit. Not also fixes #375.

**Pull request merged**